### PR TITLE
chore: update codespace for tutorial and add startup notebook

### PR DESCRIPTION
**Pull request by cpcloud**
- **chore(env): fill out devcontainer**
- **chore: add startup notebook**

### CODE REVIEW

File: 00 - Start Here.qmd [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Welcome to the Ibis tutorial!
+
+On the left 👈 is a file browser open with the tutorial notebooks.
```
Comment (ncclementi) [inline 00 - Start Here.qmd]:
```suggestion
On the left 👈 there is a file browser open with the tutorial notebooks.
```

**Comment by cpcloud** [comment]
Ok, I am pretty happy with the devcontainer setup for now. Working as expected locally. Trying on a codespace now.

**Comment by cpcloud** [comment]
Ok, the features-based version of postgres doesn't seem to work out of the box in GitHub Codespaces. Trying the compose version now.

**Comment by cpcloud** [comment]
Ok, this is working locally and on GitHub Codespaces. I'm pretty happy with the dev setup, happy to adjust the `00 - Start Here.qmd` notebook as needed

**Review by gforsyth — APPROVED** [review]
NICE

**Comment by cpcloud** [comment]
Merging so I can see whether codespace startup time is reduced after a prebuild!

**Comment by cpcloud** [comment]
Prebuild happening here: https://github.com/ibis-project/ibis-tutorial/actions/runs/8987137671/job/24684876806

**Pull request merged**